tests/library/catalog_test.py

```diff
@@ -118,15 +118,19 @@ def test_get_catalog_id_format(self, mock_get, mock_post):
 
         with self.assertRaisesRegexp(
             AssertionError,
-            "'catalog_id' must have format "
-            + r"'\/api\/v1\/catalog\/\[0-9\]\+'",
+            (
```

Comment:
Was this done to make it more readable or did a tool do this automatically?

Reply:
I did this to resolve an issue reported by flake.